public/css/transcription.css

```diff
@@ -86,6 +86,10 @@ tei-div:not(:first-of-type)>tei-p {
   text-indent: 2rem;
 }
 
+tei-div:not(:first-of-type)>tei-notatedmusic+tei-p {
```

Comment:
Should we apply this rule, or rather fix the encoding? 
In the print the paragraphs continue after the examples.

Reply:
I'm not sure. The reason for having `<notatedMusic>` outside the paragraphs was to have only the facsimile region of the  `<notatedMusic>` displaying when hovering on it, otherwise it will also display all the facsimile of the text before and after. On the other hand, the TEI encoding should be correct. 
But for now, let's apply this.